src/test/java/org/jclouds/vcloud/director/v1_5/internal/BaseVCloudDirectorApiLiveTest.java

```diff
@@ -603,31 +610,34 @@ protected void assertTaskDoneEventually(Task task) {
    /**
     * Instantiate a {@link VApp} in a {@link Vdc} using the {@link VAppTemplate} we have configured
     * for the tests.
-    * 
+    *
     * @return the VApp that is being instantiated
     */
-   protected VApp instantiateVApp() {
-      return instantiateVApp(name("test-vapp-"));
-   }
-
-   protected VApp instantiateVApp(String name) {
-//      InstantiateVAppTemplateParams instantiate = InstantiateVAppTemplateParams.builder().name(name).notDeploy()
-//               .notPowerOn().description("Test VApp").instantiationParams(instantiationParams())
-//               .source(Reference.builder().href(lazyGetVAppTemplate().getHref()).build()).build();
-
-      VdcApi vdcApi = api.getVdcApi();
-      vappId = emptyToNull(System.getProperty("test." + provider + ".vapp-id"));
-      VApp vAppInstantiated = api.getVAppApi().get(vappId);
-      assertNotNull(vAppInstantiated, String.format(ENTITY_NON_NULL, VAPP));
-
-      Task instantiationTask = getFirst(vAppInstantiated.getTasks(), null);
-      if (instantiationTask != null) {
-         assertTaskSucceedsLong(instantiationTask);
-      }
-      // Save VApp name for cleanUp
-      vAppNames.add(vAppInstantiated.getName());
-
-      return vAppInstantiated;
+   protected VApp composeVApp() {
```

Comment:
I think the javadoc of  this method is stating something different: the idea is to use a property called `test.provider.vapp-id` and instantiate it

I'd probably keep the `instantiateVApp` method and add `composeVapp` if really needed. wdyt?